jstests/core/db_level_slowms.js

```diff
@@ -0,0 +1,24 @@
+(function() {
+    "use strict";
+
+    let SlowDB = db.getSiblingDB('slowdb20171229');
+    assert.commandWorked(SlowDB.dropDatabase());
+    SlowDB.setProfilingLevel(1, {slowmsDb: 0});
+    assert(SlowDB.system.profile.count() < 2, "db.system.profile is not empty as the beginging");
```

Comment:
What is this error message supposed to be?

Reply:
hi @asya999 :  If we set the `slowmsDb` to 0, suppose every query should be logged.  so I count the number of slow query at the beginning s1, run the  10 test queries, count the number of slow query again s2.

then s2 >= 10.  

I think all of us has spent on this small feature too long time,  do you have any suggestion to make it faster?